lib/Transforms/Scalar/ScalarReplAggregatesHLSL.cpp

```diff
@@ -5896,6 +5896,16 @@ void SROA_Parameter_HLSL::createFlattenedFunction(Function *F) {
   }
 
   // Function Attr and Parameter Attr.
+  // Remove sret first.
+  if (F->hasStructRetAttr())
+    F->removeFnAttr(Attribute::StructRet);
+  for (Argument &arg : F->args()) {
+    if (arg.hasStructRetAttr()) {
+      AttributeSet SRetAS = AttributeSet::get(Ctx, arg.getArgNo() + 1, {Attribute::StructRet});
```

Comment:
This line causes this compilation error:
error C2668: 'llvm::AttributeSet::get': ambiguous call to overloaded function